kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaCommandDecoder.java

```diff
@@ -86,7 +86,7 @@ public void channelWritabilityChanged(ChannelHandlerContext ctx) throws Exceptio
             // set auto read to true if channel is writable.
             ctx.channel().config().setAutoRead(true);
         } else {
-            log.warn("channel is not writable, disable auto reading for back pressing");
+            log.debug("channel is not writable, disable auto reading for back pressing");
```

Comment:
The change that is unrelated to the PR should be put in a new PR, no matter how minor it is.

Reply:
Lots of useless warn log when testing, I will fix it in another issue.